docs/cli-reference/dfx-cycles.md

```diff
@@ -23,6 +23,52 @@ To view usage information for a specific subcommand, specify the subcommand and
 
 `dfx cycles balance --help`
 
+## dfx cycles approve
+
+Use the `dfx cycles approve` command to approve a principal to spend cycles on your behalf.
+
+### Basic usage
+
+``` bash
+dfx cycles approve [options] <spender> <amount>
+```
+
+### Arguments
+
+You must specify the following arguments for the `dfx cycles transfer` command.
+
+| Argument   | Description                           |
+|------------|---------------------------------------|
+| `<spender>`| Allow this principal to spend cycles. |
+| `<amount>` | The number of cycles to approve.      |
+
+### Options
+
+You can specify the following options for the `dfx cycles transfer` command.
+
+| Option                              | Description                                                                            |
+|-------------------------------------|----------------------------------------------------------------------------------------|
+| `--created-at-time <timestamp>`     | Specify the timestamp-nanoseconds for the `created_at_time` field on the transfer request. Useful for controlling transaction-de-duplication. https://internetcomputer.org/docs/current/developer-docs/integrations/icrc-1/#transaction-deduplication- |
+| `--expected-allowance <cycles>`     | The number of previously approved cycles.                                              |
```

Comment:
We should probably link to the [ICRC-2 specification](https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-2/README.md) in this document and even the `--help` output for this command.  At least for me, even with the description here, it's not clear what the `expected-allowance` value means.